modules/tile-converter/src/converter-cli.ts

```diff
@@ -132,7 +135,22 @@ async function convert(options) {
 // OPTIONS
 
 function parseOptions(args) {
-  const opts = {
+  const opts: {
```

Comment:
Nit: Define a `TileConversionOptions` type?

Reply:
It leads to significant refactoring of this file. I'll create another PR.